_docs/_latest/configure/scanner/rules.md

```diff
@@ -702,16 +704,17 @@ rules:
 
 * `resource`
   * `type`
-    * **Description**: The type of the resource.
+    * **Description**: The type of the resource the applies_to resources belong to.
     * **Valid values**: One of `organization`, `folder` or `project`.
 
   * `resource_ids`
     * **Description**: A list of one or more resource ids to match.
     * **Valid values**: String.
 
 * `applies_to`
-  * **Description**: A list of resource types to apply this rule to.
-  * **Valid values**: Currently only supports `bucket`.
+  * `type`
+    * **Description**: The type of resource to apply the rule to.
+    * **Valid values**: Currently only supports `bucket`.
```

Comment:
Can you also add `resource_ids` field under `applies_to`?

Reply:
Good catch, done.